rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -156,6 +157,12 @@ components:
           type: string
           format: url
           example: https://www.example.com/dataset/30f14c6c1fc85cba12bfd093aa8f90e3/file
+        metacardLocation:
+          description: >
+            This URL must support HTTP GET to retrieve the metacard.
+          type: string
+          format: url
+          example: https://www.example.com/dataset/30f14c6c1fc85cba12bfd093aa8f90e3/metacard
```

Comment:
❓ Should we update these to be UUIDs?

Reply:
I guess only the DatasetId path param matters.

Reply:
I'll create another PR right now to update both the datasetId and the examples.

Reply:
https://github.com/connexta/ion-index-api/pull/37